public/components/map_container/map_container.tsx

```diff
@@ -161,6 +161,11 @@ export const MapContainer = ({
     return () => clearInterval(intervalId);
   }, [refreshConfig]);
 
+  // Update data layers when global filter is updated
+  useEffect(() => {
```

Comment:
Do we need this hook? I think [here](https://github.com/opensearch-project/dashboards-maps/blob/06532c2d4cba833e250d660bd675eee4fa55d443/public/components/map_container/map_container.tsx#L182) will also be triggered since spatialMetaFilters prop updating is come from layer config change